perfetto-exp/main.py

```
@@ -0,0 +1,128 @@
+        desc.uuid = track_uuid
+        desc.name = name
+        if parent_track_uuid:
+            desc.parent_uuid = parent_track_uuid
+
+    # Helper to add a slice event
+    def add_slice_event(ts, event_type, event_track_uuid, name=None):
+        packet = builder.add_packet()
+        packet.timestamp = ts
+        packet.track_event.type = event_type
+        packet.track_event.track_uuid = event_track_uuid
+        packet.track_event.trac
```

Ruff (0.14.10)

Found useless expression. Either assign it to a variable or remove it.

(B018)